src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10462,20 +10466,41 @@ private ExpressionSyntax ParseTermWithoutPostfix(Precedence precedence)
                 case SyntaxKind.InterpolatedStringToken:
                     return this.ParseInterpolatedStringToken();
                 case SyntaxKind.OpenParenToken:
-                    return this.ParseCastOrParenExpressionOrLambdaOrTuple(precedence);
+                    if (IsPossibleLambdaExpression(precedence))
+                    {
+                        if (this.TryParseLambdaExpression() is { } lambda)
```

Comment:
nit: maybe merge two if's?

Reply:
i personally like the merging plus surrounding `{}`.  but i agree it is all the same anyways.  so no bggie.